Jenkinsfile

```diff
@@ -185,7 +185,7 @@ stage('Configure') {
 					// Make sure to remove that argument as soon as possible
```

Comment:
I didn't move this comment as ORM states `11, 17 or 21` Java compatibility 😃

Reply:
I'm afraid the website is out of date. Or at least, for our purposes, ORM supports JDK 22, even if that's not official.

You can certainly move the comment one line down to JDK 23.